lib/pusher/push_notifications/client.rb

```diff
@@ -40,15 +55,20 @@ def post(resource, payload = {})
       attr_reader :config
       def_delegators :@config, :instance_id, :secret_key
 
-      def build_url(resource)
-        "https://#{instance_id}.#{BASE_URL}/#{instance_id}/#{resource}"
+      def build_publish_url(resource)
+        "https://#{instance_id}.pushnotifications.pusher.com/publish_api/v1/instances/#{instance_id}/#{resource}"
+      end
+
+      def build_users_url(resource)
+        "https://#{instance_id}.pushnotifications.pusher.com/user_api/v1/instances/#{instance_id}/#{resource}"
```

Comment:
It's no longer `user_api` it's `customer_api`.

Reply:
Fixed in 14bddc4.